test/subset/generate-expected-outputs.py

```diff
@@ -35,7 +35,6 @@ def generate_expected_output(input_file, unicodes, profile_flags, instance_flags
 			"--no-overlap-flag",
 			"--no-recalc-bounds",
 			"--no-recalc-timestamp",
-			"--no-harfbuzz-repacker", # disable harfbuzz repacker so we aren't comparing to ourself.
```

Comment:
Did you mean to remove this line?

Reply:
Yes, the fonttools instancer doesn't support that flag.